### ISSUE TITLE
Error while querying Illumina HiSeq 

### ISSUE THREAD

**Issue by haripazhayath** [BioinformaticsFMRP/TCGAbiolinks#29]
I  got the error while running the command. Please help me me to fix it

query <- GDCquery(project = "TCGA-BRCA", 
                  data.category = "Gene expression",
                  data.type = "Gene Expression Quantification",
                  experimental.strategy = "RNA-seq",
                  platform = "Illumina HiSeq",
                  file.type = "results",
                  legacy = TRUE)
Error in GDCquery(project = "TCGA-BRCA", data.category = "Gene expression",  : 
  Please set a valid data.type argument from the list below:
   =>

**Comment by tiagochst** [comment]
Hi, 
I'm sorry, there was typo error. It is RNA-Seq" instead of RNA-seq. We fixed that in the vignette, but it is not updated in biocondutor yet (we pushed the code, but the vignette was not updated there it)
It is updated in the github vignette
https://github.com/BioinformaticsFMRP/TCGAbiolinks/blob/master/vignettes/tcgaBiolinks.Rmd

The correct code would be:

library(TCGAbiolinks)
query.exp <- GDCquery(project = "TCGA-BRCA", 
                      legacy = TRUE,
                      data.category = "Gene expression",
                      data.type = "Gene expression quantification",
                      platform = "Illumina HiSeq", 
                      file.type = "results",
                      experimental.strategy = "RNA-seq")

Also, we fixed the message. It will be available in Bioconductor soon.
